src/Files.App/ResourceDictionaries/PropertiesStyles.xaml

```diff
@@ -23,6 +23,8 @@
 	<Style x:Key="PropertyName" TargetType="TextBlock">
 		<Setter Property="Margin" Value="{StaticResource PropertyNameMargin}" />
```

Comment:
You kept these margins. I probably would have removed them. Are they really needed?